astropy/units/tests/test_quantity_non_ufuncs.py

```diff
@@ -2293,6 +2337,34 @@ def test_linalg_diagonal(self):
         def test_linalg_trace(self):
             self.check(np.trace)
 
+        @needs_array_function
+        def test_cross(self):
+            q1 = np.array([1, 2, 3]) << u.m
+            q2 = np.array([4, 5, 6]) << u.s
+            assert_array_equal(np.cross(q1, q2), np.linalg.cross(q1, q2))
+
+        @needs_array_function
+        def test_outer(self):
+            q = self.q.flatten()
+            assert_array_equal(np.outer(q, q), np.linalg.outer(q, q))
+
+        # @pytest.mark.skipif(True, reason="SciPy dev isn't up to date with NumPy dev")
+        @needs_array_function
+        @pytest.mark.skipif(not HAS_SCIPY, reason="Requires scipy")
+        def test_svdvals(self):
+            from scipy.linalg import svdvals as scipy_svdvals
```

Comment:
I think it should not be necessary to use `scipy.linalg` - just run  `_, ref, _ = np.linalg.svd(...)` to only keep the values.

Reply:
done !